components/build-service/base/build-pipeline-config/build-pipeline-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: build-pipeline-config
+  namespace: build-service
+data:
+config.yaml: |
+  pipelines:
+  - name: fbc-builder
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-fbc-builder:032a8745d43a942a247f365fc890b06023ccd67d
+  - name: docker-builder
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-docker-build:032a8745d43a942a247f365fc890b06023ccd67d
+  - name: java-builder
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-java-builder:032a8745d43a942a247f365fc890b06023ccd67d
+  - name: nodejs-builder
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-nodejs-builder:032a8745d43a942a247f365fc890b06023ccd67d
```

Comment:
Did we have a decision on whether we were creating these?

Reply:
I think we can omit s2i since they are not used, AFAIK.

Reply:
removed